plugins/outputs/wavefront/README.md

```diff
@@ -49,6 +49,12 @@ This plugin writes to a [Wavefront](https://www.wavefront.com) proxy, in Wavefro
   ## Truncate metric tags to a total of 254 characters for the tag name value. Wavefront will reject any 
   ## data point exceeding this limit if not truncated. Defaults to 'false' to provide backwards compatibility.
   #truncate_tags = false
+
+  ## Flush the internal buffers after each batch. This effectively bypasses the background sending of metrics
+  ## normally done by the Wavefront SDK. This can be used if you are experiencing buffer overruns. The sending 
+  ## of metrics will block for a longer time, but this will be handled gracefully by the internal buffering in
+  ## Telegraf.
+  #immediate_flush = false
```

Comment:
I don't think we should provide this as an option, and instead just always assume it's true.

Reply:
I would agree, but I don't want to break backwards compatibility.

Reply:
Seems like it was already broken. Maybe I'm missing something. Is the plugin in use elsewhere?

Reply:
It's in use with hundreds of customers. I don't have a way of testing the behavior in an extremely busy environment and with heavily overloaded connections. I'm not comfortable changing the behavior without a way to roll back to the current behavior. Can we let it be a flag for now and deprecate it in the next release if we don't see any issues?

Reply:
I see what you're getting at; I'm willing to leave it in temporarily. I'd like to default it to `true`, because I don't believe the `false` behavior works correctly for anyone.

Reply:
Done. Can we merge this now?